pages/History.qml

```diff
@@ -34,342 +34,1519 @@ import moneroComponents.WalletManager 1.0
+                                    label.font.family: FontAwesome.fontFamily
+                                    fontSize: 18 * scaleRatio
+                                    width: 36 * scaleRatio
+
+                                    MouseArea {
+                                        state: "proof"
+                                        anchors.fill: parent
+                                        hoverEnabled: true
+                                        z: parent.z + 1
+                                    }
+                                }
+                            }
+                        }
+                    }
+
+                    ColumnLayout {
+                        spacing: 0
+                        Layout.fillWidth: true
+                        Layout.preferredHeight: 40
+
+                        Rectangle {
+                            color: "transparent"
+                            Layout.fillWidth: true
+                            Layout.preferredHeight: 20
+
+                            Text {
+                                font.family: MoneroComponents.Style.fontRegular.name
+                                font.pixelSize: 15 * scaleRatio
+                                text: "Description"
+                                color: "#C0C0C0"
+                                anchors.verticalCenter: parent.verticalCenter
+                            }
+                        }
+
+                        Rectangle {
+                            color: "transparent"
+                            Layout.fillWidth: true
+                            Layout.preferredHeight: 20
+
+                            Text {
+                                id: txNoteText
+                                font.family: MoneroComponents.Style.fontRegular.name
+                                font.pixelSize: 15 * scaleRatio
+                                text: tx_note !== "" ? tx_note : "-"
+                                color: MoneroComponents.Style.defaultFontColor
+                                anchors.verticalCenter: parent.verticalCenter
+
+                                MouseArea {
+                                    state: "copyable"
+                                    anchors.fill: parent
+                                    hoverEnabled: true
+                                    onEntered: parent.color = MoneroComponents.Style.orange
+                                    onExited: parent.color = MoneroComponents.Style.defaultFontColor
+                                }
+                            }
+
+                            Image {
+                                anchors.top: parent.top
+                                anchors.left: txNoteText.right
+                                anchors.leftMargin: 12 * scaleRatio
+                                source: "../images/editIcon.png"
+                                opacity: 1
+                                width: 18
```

Comment:
```suggestion
                                width: 18 * scaleRatio
```

Reply:
@selsta and I have been talking, we're moving away from scaleRatio. It has absolutely no use right now. Instead, people should use `QT_SCALE_FACTOR`.